src/components/shared/Card.module.css

```diff
@@ -102,6 +102,11 @@ a.linkedContainer {
   margin-top: 30px;
 }
 
+.companyName {
```

Comment:
FYI We shouldn't ever have to style text directly like this.  Instead we can use any of the H1-H6 tags and inherit these styles.

Reply:
Hmm yeah I think I agree.  The closest relative is `H6` but it also carries with it some margin that I don't want.  Do we still add `.companyName { magin: 0 }`?

Reply:
Yep, adding spacing is totally fine in .companyName